src/utils/e2e-testing.ts

```diff
@@ -12,17 +12,27 @@ import { rootLogger } from './logger'
 import { PackageManagerType } from './package-manager'
 import stripAnsi = require('strip-ansi')
 
-const log = rootLogger.child('e2e')
+const log = rootLogger.child('e2e-testing')
 
-export function setupE2EContext(nexusOutputDir?: string) {
-  const tmpDir = nexusOutputDir ?? getTmpDir('nexus-prisma-tmp-')
-  const NEXUS_BIN_PATH = Path.join(tmpDir, 'node_modules', '.bin', 'nexus')
-  log.trace('setup', { tmpDir, NEXUS_BIN_PATH })
+export function setupE2EContext(config: {
```

Comment:
I did my best to encapsulate/model what's going on. I concluded what we're dealing with is linked mode vs not linked mode. If you read the docs for npm install it actually performs a link when doing folder installs (unless folder is within project).

After first I had env vars being set at test suite level. But that doesn't really make sense because there are rules about how linked tests work. So it makes sense that e2e-testing module encapsulates them. Then e.g. prisma plugin doesn't have to "rediscover" or "reconfigure" for these rules, etc.

The direction is then away from an e2e-testing module and toward just a system-testing module. E2E testing becomes just a mode the module can operate in, rather than the only thing the module can do.

But I haven't done big renaming yet. This is what I think. Let's see how we feel about it in a few days, weeks.